charts/harvester-vm-import-controller/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: rancher/harvester-vm-import-controller
   pullPolicy: Always
```

Comment:
pullPolicy will need be `IfNotPresent` since it is a relased version.

Reply:
thanks.. Fixed it up.